client/layout/store-alerts/index.js

```diff
@@ -104,18 +65,23 @@ class StoreAlerts extends Component {
 		const numberOfAlerts = alerts.length;
 		const alert = alerts[ currentIndex ] ? alerts[ currentIndex ] : null;
 		const type = alert && alert.type ? alert.type : null;
-		const icon = type && alertTypes[ type ] ? alertTypes[ type ].icon : null;
 		const className = classnames( 'woocommerce-store-alerts', {
-			'is-alert-emergency': 'emergency' === type,
-			'is-alert-alert': 'alert' === type,
-			'is-alert-critical': 'critical' === type,
+			'is-alert-error': 'error' === type,
+			'is-alert-update': 'update' === type,
 		} );
+		const actions =
+			alert &&
+			alert.actions.map( action => (
+				<Button key={ action.name } isDefault href={ action.url }>
```

Comment:
Is there any chance an action might not have a name? I don't remember off the top fo my head if it is required or not.

Reply:
I tried to add an action without a name and it wouldn't let me, so disregard :)